xmipptomo/protocols/protocol_fit_ellipsoid.py

```diff
@@ -60,39 +62,60 @@ def _insertAllSteps(self):
 
     # --------------------------- STEPS functions -------------------------------
     def fitEllipsoidStep(self):
-        inputSubtomos = self.inputSubtomos.get()
+        import time
+        time.sleep(10)
```

Comment:
Is this use for the protocol or is it for debugging? Probably it can be removed?

Reply:
certainly it can be. Done.